### ISSUE TITLE
Plugin not working with WebStorm 2020.1

### ISSUE THREAD

**Issue by lebolo** [denofevil/AureliaStorm#38]
Getting an error after upgrading to WebStorm 2020.1.

```
Plugin Error: Plugin "AureliaStorm" is incompatible (until build 201.0 < WS-201.6668.106).
```

I'm not sure if this plugin is still being maintained by @denofevil. Any plans on updating it for 2020.1?

I don't know any Kotlin :( but this plugin is super useful so may try to poke around. Wondering if we can get the Aurelia community to step in.

**Comment by lebolo** [comment]
A little more poking around and found this https://github.com/aurelia/tools/issues/55#issuecomment-278306033. @denofevil, can you point to some quick resources for plugin development?

**Comment by denofevil** [comment]
I've published 0.9.1.1 which fixes the compatibility issue.

As for plugin development, I'd recommend [IntelliJ Platform SDK guide](https://www.jetbrains.org/intellij/sdk/docs/intro/welcome.html). I'm not actively developing this plugin, but feel free to bug me with questions 😊
